modules/ROOT/pages/to-deprecate-asset.adoc

```diff
@@ -1,33 +0,0 @@
-= Deprecate an Asset
```

Comment:
do you want to add a page alias?

Reply:
Hi @sathya0, not this file. Thanks for checking.